b2luigi/batch/processes/gbasf2.py

```diff
@@ -416,6 +439,10 @@ def _build_gbasf2_submit_command(self):
         if gbasf2_n_repition_jobs is not False:
             gbasf2_command_str += f" --repetition {gbasf2_n_repition_jobs} "
 
+        gbasf2_input_datafiles = get_setting("gbasf2_input_datafiles", default=[], task=self.task)
```

Comment:
All settings should be documented so that people can find them when reading the documentation, otherwise the only way to find them would be to read the code.
- [ ] document new setting

Btw all gbasf2 options are in theory supported via the `gbasf2_additional_params` setting, but I'm not against adding settings for frequently used parameters for convenience.

Reply:
Is that not covered by the optional settings description in line [here](https://github.com/nils-braun/b2luigi/pull/91/files#diff-5abe42743d6a1073f12dd23f761c0cae74e8acd8b697657a494bdf56d8476bc1L157-L167)?

Reply:
Sorry overlooked that.